.github/workflows/cli-tests.yml

```
@@ -25,7 +25,7 @@ jobs:
           - name: linux
             image: ubuntu-20.04 # Focal Fossa
           - name: mac
-            image: macos-12
+            image: macos-latest
```

**Note from Author:** This may cause a problem at unknown times, but may be better than forgetting to update this through time.

---

Tests failed on `macos-latest` (presumably `macos-15`, so trying with `macos-14`.
```suggestion
            image: macos-14
```